gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -262,6 +264,20 @@ private boolean isDirectPathEnabled(String serviceAddress) {
     return false;
   }
 
+  private boolean isDirectPathXdsUsed() {
+    // Method 1: Enable DirectPath xDS by env.
+    String directPathXdsEnv = envProvider.getenv(DIRECT_PATH_ENV_ENABLE_XDS);
+    boolean isDirectPathXdsEnv = Boolean.parseBoolean(directPathXdsEnv);
+    if (isDirectPathXdsEnv) {
+      return true;
+    }
+    // Method 2: Enable DirectPath xDS by option.
+    if (Boolean.TRUE.equals(useDirectPathXds)) {
```

Comment:
Usually an explicitly set value has higher priority than environment variable, otherwise the users of this public field could be confused why their explicitly set value does not work. 
That being said, this class and the whole DirectPath feature is supposed to be use by internal users only, so I'm fine with it if it's a requirement that env takes precedence over options, at least we should document this behavior in Javadoc.